igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -49,7 +52,7 @@ class BuildController {
     ExecutorService executor
 
     @Autowired
-    BuildMasters buildMasters
+    BuildServices buildMasters
```

Comment:
Maybe change the variable name to match the type?

Reply:
Pushed a new commit to rename this everywhere

Reply:
:+1: